org.gnome.Fragments.json

```diff
@@ -0,0 +1,48 @@
+{
+    "app-id" : "org.gnome.Fragments",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "fragments",
+    "finish-args" : [
+        "--filesystem=home",
+        "--device=dri",
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+	"--filesystem=xdg-run/dconf",
+	"--filesystem=~/.config/dconf:ro",
+	"--talk-name=ca.desrt.dconf",
+	"--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "build-args" : ["--share=network"]
```

Comment:
We disallow network access during build for at least basic reproducibility.

Reply:
Hm that's bad, since I use nested git submodules, which needs to be downloaded.

Reply:
Nested git submodules should be downloaded fine by git itself. Just the build process is offline, sources are downloaded before that.